src/containers/Chat/ChatMessages/ChatInput/ChatInput.tsx

```diff
@@ -100,7 +100,7 @@ export const ChatInput: React.SFC<ChatInputProps> = (props) => {
   }
 
   const submitMessage = (message: string) => {
-    if (!message) return;
+    if (!message || message === ' \n') return;
```

Comment:
If we add a space and then press enter, the message is still sent.
Logic should be, in my view, "remove space and check the length."